src/arrayMethodSort.js

```diff
@@ -3,9 +3,28 @@
 /**
  * Implement method Sort
  */
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) =>
+    a.toString() > b.toString()) {
```

Comment:
Edit your compare function, it should also return `-1`.

For more details you could  [rewatch video](https://mate.academy/learn/javascript-advanced/array-iteration-methods#/video/152)

Reply:
Thanks for the feedback, made changes according to the comments))